src/connection.h

```diff
@@ -145,6 +145,7 @@ struct connListener {
     int port;
     ConnectionType *ct;
     void *priv; /* used by connection type specified data */
+    char *group; /* used to specify unix socket group */
```

Comment:
i see we put unixsocketperm into priv, and here we are adding a new group field? i suppose it is ok since the code is easy to write/read.

Reply:
I was also unsure about this. The other idea I had was to make it priv1/priv2 to just keep it simple. I don't really want to allocate extra memory just to put in a private field.

Reply:
> I don't really want to allocate extra memory just to put in a private field.

yes, i agree. let's use priv1/priv2